modules/server_iam/main.tf

```diff
@@ -0,0 +1,101 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  random_hash    = var.suffix
+  server_sa_name = "forseti-server-gcp-${local.random_hash}"
+
+  server_project_roles = [
+    "roles/storage.objectViewer",
+    "roles/storage.objectCreator",
+    "roles/cloudsql.client",
+    "roles/cloudtrace.agent",
+    "roles/logging.logWriter",
+    "roles/iam.serviceAccountTokenCreator",
+  ]
+  server_write_roles = [
+    "roles/compute.securityAdmin",
+  ]
+  server_read_roles = [
+    "roles/appengine.appViewer",
+    "roles/bigquery.metadataViewer",
+    "roles/browser",
+    "roles/cloudasset.viewer",
+    "roles/cloudsql.viewer",
+    "roles/compute.networkViewer",
+    "roles/iam.securityReviewer",
+    "roles/orgpolicy.policyViewer",
+    "roles/servicemanagement.quotaViewer",
+    "roles/serviceusage.serviceUsageConsumer",
+  ]
+  server_bucket_roles = [
+    "roles/storage.objectAdmin",
+  ]
+  server_cscc_roles = [
+    "roles/securitycenter.findingsEditor",
+  ]
+}
+
+#-------------------------#
+# Forseti Service Account #
+#-------------------------#
+resource "google_service_account" "forseti_server" {
+  account_id   = local.server_sa_name
+  project      = var.project_id
+  display_name = "Forseti Server Service Account"
+}
+
+resource "google_project_iam_member" "server_roles" {
+  count   = length(local.server_project_roles)
+  role    = local.server_project_roles[count.index]
+  project = var.project_id
+  member  = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_organization_iam_member" "org_read" {
+  count  = var.org_id != "" ? length(local.server_read_roles) : 0
+  role   = local.server_read_roles[count.index]
+  org_id = var.org_id
+  member = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_folder_iam_member" "folder_read" {
+  count  = var.folder_id != "" ? length(local.server_read_roles) : 0
+  role   = local.server_read_roles[count.index]
+  folder = var.folder_id
+  member = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_organization_iam_member" "org_write" {
+  count  = var.org_id != "" && var.enable_write ? length(local.server_write_roles) : 0
+  role   = local.server_write_roles[count.index]
+  org_id = var.org_id
+  member = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_folder_iam_member" "folder_write" {
+  count  = var.folder_id != "" && var.enable_write ? length(local.server_write_roles) : 0
+  role   = local.server_write_roles[count.index]
+  folder = var.folder_id
+  member = "serviceAccount:${google_service_account.forseti_server.email}"
+}
+
+resource "google_organization_iam_member" "org_cscc" {
+  count  = var.org_id != "" && var.cscc_violations_enabled ? length(local.server_cscc_roles) : 0
```

Comment:
Can you change the `cscc_violations_enabled` in `variables.tf` to be `type = bool`?